Nos/Views/ActionButton.swift

```diff
@@ -3,9 +3,15 @@ import SwiftUI
 /// A big bright button that is used as the primary call-to-action on a screen.
 struct ActionButton: View {
     
+    enum ImageAlignment {
+        case left
```

Comment:
I think to be i18n localized these should be "leading" and "trailing" and we should reverse them in RTL languages. But we don't support any RTL languages right now so I'm not that worried about it.

Reply:
I think that would make sense. However, I expected the copy icon to be at the left honestly, even for English or Spanish, but it's in the right in the design. And this made me realize that the position of the icon, in the Figma design, doesn't depend on the locale. But I'm open to add this, it seems to be an interesting problem to solve as a programmer.